hibernate-core/src/test/java/org/hibernate/orm/test/annotations/inheritance/joined/JoinedSubclassAndSecondaryTable.java

```diff
@@ -155,16 +155,17 @@ public void testSecondaryTableAndJoinedInverse(SessionFactoryScope scope) {
 	private void assertNumberOfRows(Session s, String tableName, int expectedNumberOfRows, String message) {
 		s.doWork(
 				work -> {
-					PreparedStatement preparedStatement = work.prepareStatement( "select count(*) as row_count from " + tableName );
-					preparedStatement.execute();
-					ResultSet resultSet = preparedStatement.getResultSet();
-					resultSet.next();
-					long rowCount = resultSet.getLong( "row_count" );
-					assertEquals(
-							expectedNumberOfRows,
-							rowCount,
-							message
-					);
+					try (PreparedStatement preparedStatement = work.prepareStatement( "select count(*) as \"row_count\" from " + tableName )) {
```

Comment:
Does this work on MySQL?

Reply:
Good point; I'll replace it with something that is not a keyword in either.

Reply:
thanks